src/engine/renderer/tr_image.cpp

```diff
@@ -143,6 +143,417 @@ void GL_TextureMode( const char *string )
 	}
 }
 
+int GetFormatPixelSize( GLenum internalFormat )
```

Comment:
In Memory, I can rename it right. Also, it can't return perfect answer in all case, for example it is somewhat wrong with DXT textures when a picture has less than 4 pixels (maybe I can add a comment about that).

Reply:
Added comments about it.